dev/create-release/spark-rm/Dockerfile

```diff
@@ -76,10 +75,8 @@ RUN apt-get clean && apt-get update && $APT_INSTALL gnupg ca-certificates && \
   Rscript -e "install.packages(c('curl', 'xml2', 'httr', 'devtools', 'testthat', 'knitr', 'rmarkdown', 'roxygen2', 'e1071', 'survival'), repos='https://cloud.r-project.org/')" && \
   Rscript -e "devtools::install_github('jimhester/lintr')" && \
   # Install tools needed to build the documentation.
-  $APT_INSTALL ruby2.3 ruby2.3-dev mkdocs && \
-  gem install jekyll --no-rdoc --no-ri -v 3.8.6 && \
-  gem install jekyll-redirect-from -v 0.15.0 && \
-  gem install rouge
+  $APT_INSTALL ruby2.5 ruby2.5-dev && \
```

Comment:
So, did `ruby2.5` work correctly?

Reply:
`mkdocs` requires ruby 2.4+, I googled a bit and seems ubuntu 18 supports ruby 2.5 at most.